windows/sidecar.ps1

```diff
@@ -126,10 +126,10 @@ if ($networkConfigObj)
                 }
                 "host-gw" {
                     $networkAddress = ""
-                    if ($nodeAddress)  {
-                        $networkAddress = $nodeAddress
-                    } elseif ($nodeInternalAddress) {
+                    if ($nodeInternalAddress) {
                         $networkAddress = $nodeInternalAddress
+                    } elseif ($nodeAddress)  {
+                        $networkAddress = $nodeAddress
```

Comment:
Does it change the current? seems Irrelevant

Reply:
Oh, these changes are for solving the followed case: when indicating both `address` and `internal address`, the below code (`wins.exe cli net get --address "$networkAddress"`) could not detect a network adapter which addressed with VPC public IP :
https://github.com/rancher/rke-tools/blob/29b2108f901470e3e54f3a53223a4272bd80fd56/windows/sidecar.ps1#L130-L137

So I swap to use the `internal address` to detect the truly existing network adapter.

Reply:
Got it.